src/webview/view/components/forms/spfxProject/AdditionalStep.tsx

```diff
@@ -154,6 +158,15 @@ export const AdditionalStep: React.FunctionComponent<AdditionalStepProps> = ({
                         value={shouldCreateNodeVersionFile}
                         setValue={setShouldCreateNodeVersionFile}
                         label='Create node version manager configuration file' />}
+                {shouldInstallCustomSteps &&
+                    <label className={'block mb-1'}>
+                    To configure this setting, go to **Settings** (`Ctrl + ,`) and expand **Extensions**. Then navigate to **SharePoint Framework Toolkit** and scroll to the **`Spfx-toolkit.ProjectCustomSteps`** setting.
```

Comment:
let's make the explanation more generic also since on MacOs there is no `Ctrl`  so this label would not apply.
```suggestion
                    You may modify the custom install step in the extension settings.
```